source/start/topics/tutorials/config_pitfalls.rst

```diff
@@ -151,11 +151,11 @@ GOOD:
         }
     }
 
-Using If
+Using ``if``
 --------
 
```

Comment:
The dashes need to go under the full length of the title, the backtick markup changes means it needs to be four characters longer.